node/core/backing/src/lib.rs

```diff
@@ -190,6 +190,7 @@ struct AttestingData {
 	backing: Vec<ValidatorIndex>,
 }
 
+/// How many votes we need to consider a candidate backed.
```

Comment:
could you add a comment that it will be changed soon
> // This can only be meaningfully enacted once the changed limit in the runtime is live.

Reply:
That also means we need a synchronous runtime upgrade on Polkadot and Kusama?

Reply:
Not necessarily synchronous, but it only makes sense to update node code once both have upgraded.